authors/apps/authentication/tests/test_api.py

```diff
@@ -30,47 +30,6 @@ def test_register_user(self):
         self.assertEqual(result["user"]["email"], "caro@yahoo.com")
         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
 
-    def test_register_user_existing_email(self):
```

Comment:
Do we need to remove these tests?

Reply:
These tests were duplicated in test_registration_error_messages.py